sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/src/test/resources/application.yaml

```diff
@@ -0,0 +1,27 @@
+spring:
+  main:
+    banner-mode: off
+  cloud:
+    azure:
+      eventhub:
+        connection-string: ${EVENTHUB_CONNECTION_STRING_BINDER}
```

Comment:
We could the same event hub namespace and storage account, but a different event hub and storage container.